react/marketplace/marketplace/src/index.tsx

```diff
@@ -8,7 +8,7 @@ import { Provider } from 'react-redux';
 
 import store from './shared/store/store';
 
-import AppContainer from './components/App/AppContainer';
+import AppContainer from './components/app/AppContainer';
```

Comment:
it's weird that the directory is lowercase. can't we uppercase the dir instead? that's the convention for Angular components i think. good find nevertheless, this would break on linux/bsd

might need two commits for git to detect the change btw, `app -> BLAH -> APP`.

Reply:
I believe it was uppercase previously - but then it caused compilation errors on my OSX. Good suggestion about the two commit process though, I'll look at doing this.

Reply:
yeah, according to GH, the dirname is `app`, not `App`. that may be the deeper cause.

if you rename the dir, then you should check whether any other files load things from "app", ofc.